app/assets/stylesheets/application.scss

```diff
@@ -1,2 +1,276 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "font-awesome";
+
+$teal: #159E98;
+$other-highlight: #218578;
+$teal-grey: #32A69B;
+$dark-teal: #238479;
+$rust: #C18763;
+$yellow: #D6D8A6;
+
+$light-contrast: $teal-grey;
+$flash: #75E8E1;
+$highlight: $teal;
+$neutral: $yellow;
+
+/*$body: 'IM Fell DW Pica';*/
+$body: 'Tinos';
+$header: 'Tinos';
+
+.each-merchant {
+  padding-bottom: 70px;
+}
+
+.merchant-name {
+  padding-bottom: 20px;
+  text-align: center;
+  max-width: 380px;
+}
+
+.centerBlock {
+  display: table;
+  margin: 0 auto;
+  margin-top: 20px;
+  margin-bottom: 20px;
+            }
+
+.product-names {
+  text-align: center;
+}
+
+.emptycart {
+  text-align: center;
```

Comment:
When you're using the same CSS property for multiple classes or ids, you can add them together like so:
```css
.product-names, .emptycart, .flash {
  text-align: center
}
```